### PR TITLE
81523 App allows create an employee with only a space in the Name (without any additional characters)

### PR DESCRIPTION
**Pull request by lukino563**

### CODE REVIEW

File: server/ERNI.PBA.Server.Business/Handlers/Users/CreateUserHandler.cs [hunk 1]
```diff
@@ -35,6 +36,11 @@ public async Task<bool> Handle(CreateUserCommand request, CancellationToken canc
                 throw new OperationErrorException(StatusCodes.Status409Conflict);
             }
 
+            if (request.FirstName.Trim() == String.Empty || request.LastName.Trim() == String.Empty)
```
Comment (jakubsturc) [inline server/ERNI.PBA.Server.Business/Handlers/Users/CreateUserHandler.cs:39]: Consider to use String.IsNullOrWhiteSpace method

File: client/src/app/users/create-user/create-user.component.ts [hunk 2]
```diff
@@ -73,27 +73,15 @@ export class CreateUserComponent implements OnInit {
...
-        } else {
-            this.controls.firstName.setErrors({'required': false});
-        }
-
-        if (!this.controls.lastName.value.trim()) {
-            this.controls.lastName.setErrors({'required': true});
-        } else {
-            this.controls.lastName.setErrors({'required': false});
-        }
+    trimOnBlur() {
```
Comment (jakubsturc) [inline client/src/app/users/create-user/create-user.component.ts:76]: - Maybe you can introduce the control whose value shall be trimmed as a parameter of the method.
- I am not sure about naming. `OnBlur` part of the name is telling when it is done, not what is done.

File: server/ERNI.PBA.Server.Business/Handlers/Users/CreateUserHandler.cs [hunk 3]
```diff
@@ -36,15 +36,15 @@ public async Task<bool> Handle(CreateUserCommand request, CancellationToken canc
...
                 throw new OperationErrorException(StatusCodes.Status400BadRequest);
             }
 
             var user = new User
             {
-                FirstName = request.FirstName,
-                LastName = request.LastName,
+                FirstName = request.FirstName.Trim(),
+                LastName = request.LastName.Trim(),
                 Username = request.Email,
```
Comment (jakubsturc) [inline server/ERNI.PBA.Server.Business/Handlers/Users/CreateUserHandler.cs:48]: We don't need to trim email address (username), do we?

Comment (lukino563) [reply server/ERNI.PBA.Server.Business/Handlers/Users/CreateUserHandler.cs:48]: Yes, because we can't have trailing or leading whitespaces in name

**Pull request merged**